docs/content/latest/develop/learn/ttl-data-expiry.md

```diff
@@ -1,30 +1,47 @@
 ---
-title: TTL for data expiry
-linkTitle: TTL for data expiry
+title: 9. TTL for data expiry
+linkTitle: 9. TTL for data expiry
+description: TTL for data expiry
 aliases:
   - /develop/learn/ttl-data-expiry/
+menu:
+  latest:
+    identifier: ttl-data-expiry
+    parent: learn
+    weight: 581
+showAsideToc: true
+isTocNested: true
 ---
+In YCQL there are two types of TTL, the table TTL and column level TTL. The column TTLs are stored
+with the value using the same encoding as Redis. The Table TTL is not stored in DocDB (it is stored
```

Comment:
Table -> table

Reply:
Why is "using the same encoding as Redis" useful or relevant in the docs here?

Reply:
It was old description. removing.